align 'Applies-to' and actions

tophf opened the pull request:
* hid the `Applies-to` label and shifted (i) left, didn't change html to allow people to unhide it
* fixed select's height, it was smaller by 2px
* fixed up/down icons to show as triangles correctly
* removed the accidental margin on the right of the applies-to block

![image](https://user-images.githubusercontent.com/1310400/151461862-45553e4c-54b4-4865-94fc-7b851830aa02.png)

narcolepticinsomniac commented:
Alignment adjustments, and hiding label LGTM.

> fixed select's height, it was smaller by 2px

This is causing unintended issues everywhere, particularly the manager. I see the height was off by a couple pixels, but judging by the CSS, it's the input that's a couple pixels taller than intended. Seems it's simply missing `box-sizing: border-box;`.

If we're fixing minor CSS issues, I've noticed a couple recently. `datails` under 'editor settings' have little content jumps when toggled. Also 'style settings' dialog inputs/textareas cut off focus outlines on the sides, and the 'on save' checkbox is wedged tightly between two buttons.

tophf commented:
Fixed everything except `details` as I didn't understand what's jumping and `on save` margins as they are bigger than our standard gap between buttons and the intent here is to read it as a phrase: "Save on change". We could have increased the right margin but it would probably look unbalanced.

narcolepticinsomniac commented:
> I didn't understand what's jumping

```
#header details[open] + details[open] {
    margin-top: .5rem;
}
```

IDK what's changed, but they didn't always jump when toggled.

> they are bigger than our standard gap

Must be some confusion, as this is what I'm referring to:

![settings](https://user-images.githubusercontent.com/14100003/151468291-0d0c45fd-d24e-460f-9619-d3c26fe46d59.png)

tophf commented:
Do you mean this behavior:

![chrome_2022-01-28_12-43-50](https://user-images.githubusercontent.com/1310400/151524364-2de90c68-3497-4758-b15c-fdc0afef9902.gif)

As for the modal, it's caused by `margin-inline` that I added earlier, which MDN said is supported since Chrome 66 so either it lied or there was a bug in Chrome fixed just recently because I don't see it in versions after 95. Anyway, it was wrong to use a feature that's not supported in 55-65 range that we officially still support so I've switched to a more compatible CSS.
